test/servico/cst01Test.java

```diff
@@ -52,23 +51,25 @@ public void tearDown() {
     }
     
     @Test
-    public void testeNotificaçãoDeAusência_ÚnicoHorário() throws ParseException
-    {
+    public void testeNotificaçãoDeAusência_ÚnicoHorário() throws ParseException{
+        
+        //Login com usuário: Funcionario
         LoginService loginService = new LoginService();
-        assertTrue(loginService.VerificarUsuarioESenha("Funcionario1", "123456"));
+        assertTrue(loginService.VerificarUsuarioESenha("Funcionario", "123456"));

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 80 characters (found 81).](https://www.codacy.com/app/calebepb/substituicao/file/2738760059/issues/source?bid=3308597&fileBranchId=3353768#l58)